components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.ts

```diff
@@ -203,12 +226,38 @@ export class DataFeedCreateComponent {
         this.integrationSelected = true;
         break;
       }
+      case WebhookIntegrationTypes.CUSTOM: {
+        this.showCustomField();
```

Comment:
```suggestion
        this.showFieldWebhook();
        this.showFields.useHeaders = true;
```
We dont need the showCustomField() method as most feilds are same apart from header